docs/developer-guides/grpcrestapi.md

```diff
@@ -3,4 +3,28 @@ id: grpcrestapi
 sidebar_label: gRPC Gateway (REST API)
 hide_table_of_contents: true
 
----
\ No newline at end of file
+---
+
+# GRPC Gateway API 
+
+Babylond's implementation is based on CometBFT node rpcs. In addition, some custom modules also allow Babylon specific methods. 
+
+Please refer to both Babylon gRPC API and CometBFT API specifications for details. 
+
+[Babylon Custom gRPC](https://app.swaggerhub.com/apis-docs/jvssptyltd/babylon_g_rpc_gateway_docs/1.0.0)
+
+[Babylon Standard gRPC (CometBFT v0.83)](https://app.swaggerhub.com/apis-docs/jvssptyltd/comet-bft_rpc/v0.38.x)
+
+
+Phase 3: Devnet Endpoints
+
+| Provider      | RPC Endpoint                          | LCD Nodes Endpoint                     | gRPC Endpoint                          |
+|---------------|---------------------------------------|---------------------------------------|----------------------------------------|
+| Babylon Labs  | https://rpc.devnet.babylonlabs.io    | https://lcd.devnet.babylonlabs.io    | https://grpc.devnet.babylonlabs.io    |
+
+Phase 2: Testnet Endpoints
+
+| Provider      | RPC Endpoint                          | LCD Nodes Endpoint                     | gRPC Endpoint                          |
```

Comment:
How about we link to our [networks](https://github.com/babylonlabs-io/networks/tree/main/bbn-test-5) page to avoid having to update this if the URLs change?

Reply:
Okay, let me direct people there. I think it only makes sense to provide Phase 2 testnet endpoints for now as we don't expect a lot of activities on Phase 3 Devnet yet. 

Ideally, I would build a "network information" page to display all network parameters including RPC endpoints.